cmd/server/assets/cstatus.html

```diff
@@ -0,0 +1,164 @@
+{{define "cstatus"}}
+<!doctype html>
+<html lang="en">
+
+<head>
+  {{template "head" .}}
+
+  <style>
+    #vercode {
+      flex-shrink: 10;
+      line-height: 2.3em;
+      max-width: 480px;
+    }
+
+    #vercode span {
+      flex: 1;
+      margin: 0 0.1em;
+    }
+  </style>
+</head>
+
+<body>
+  {{template "navbar" .}}
+
+  <main role="main" class="container">
+    {{template "flash" .}}
+
+    <h1>Check verification code status</h1>
+    <p>
+      Use a unique identifier to check the status of the verification code
+      previously shared with your patient.
+    </p>
+
+    <form id="statusForm" action="#">
+      <div class="row form-group">
+        <label for="uuid" class="col-sm-3">Code Identifier</label>
+        <div class="col-sm-9">
+          <div class="input-group p-0 shadow-sm">
+            <input type="text" id="uuid" name="uuid" class="form-control" />
+          </div>
+          <small class="form-text text-muted">
+            Identifier in the format XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX given from the Issue Code form.
+          </small>
+        </div>
+      </div>
+
+      <div class="row form-group">
+        <div class="offset-sm-3 col-sm-9">
+          <button id="submit" class="btn btn-primary btn-block">Check Status</button>
+        </div>
+      </div>
+    </form>
+
+    <div id="codeerror" class="alert alert-danger d-none"></div>
+
+    <div id="codearea" class="d-none">
+      <hr>
+
+      <h2>Code status</h2>
+
+      <p>
+        The claimed status of this code is:
+      </p>
+
+      <div id="vercode"
+        class="text-center text-monospace d-flex flex-row align-content-center justify-content-around flex-nowrap mx-auto mb-3">
+      </div>
+
+      <button id="startover" class="btn btn-warning btn-block mb-3">
+        Clear and check another code
+      </button>
+    </div>
+  </main>
+
+  {{template "scripts" .}}
+
+  <script type="text/javascript">
```

Comment:
this hide/unhide div is what the issue page does. For now it might make sense to start em the same way and maybe later change both if that's what want.